tests/examples/Interaction-overlay.py

```
@@ -0,0 +1,54 @@
+
+_bootstrap_local_packages()
+
+import notte  
+
+
+def main() -> None:
+    load_dotenv()
+    #Use save_replay_to in Session method to save the screenshots WebP file at the specified location.
+    with notte.Session(headless=False, save_replay_to=r".\replays\rp.webp") as session:
+        agent = notte.Agent(session=session, reasoning_model="gemini/gemini-2.5-flash", max_steps=30)
+        response = agent.run(task="go to google and search for plujss and select the second dropdown suggestion")
```

Ruff (0.14.3)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)